### PR TITLE
MODIFIED changed min and max allowable date range. max date autoupdated

### PR DESCRIPTION
**Pull request by phoenixwhsk**
Date issues resolved

### CODE REVIEW

**Comment by raj131095** [comment]
date change range

**Review by raj131095 — COMMENTED** [review]
##hello

**Review by kalyanikp — DISMISSED** [review]
add datetime

File: mysql_project/mysql_dash/mysql_dash1/plotly_app.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def call_back(start_date,end_date,value):
       html.Div([
             html.H3('select start and end date: '),
               dcc.DatePickerRange( id='my_date_pick',
-                                   min_date_allowed= datetime.date(2019,1,1)
-                                   ,max_date_allowed=datetime.date(2021,12,31),
```
Comment (phoenixwhsk) [inline mysql_project/mysql_dash/mysql_dash1/plotly_app.py:87]: New date auto update is not enabled

Comment (raj131095) [reply mysql_project/mysql_dash/mysql_dash1/plotly_app.py:87]: let me see this

File: mysql_project/mysql_dash/mysql_dash1/plotly_app.py [hunk 2]
```diff
@@ -83,8 +83,8 @@ def call_back(start_date,end_date,value):
...
+                                   min_date_allowed= datetime.date(2019,3,19),
```
Comment (kalyanikp) [inline mysql_project/mysql_dash/mysql_dash1/plotly_app.py:86]: change min date to 1st march

Comment (phoenixwhsk) [reply mysql_project/mysql_dash/mysql_dash1/plotly_app.py:86]: 19th is the start date

**Pull request merged**